### DISCUSSION TITLE
How to detect mouse clicks? Can't resolve PointerPressed unresolved external symbol! C++/WinRT - WinUI 3

### DISCUSSION THREAD

**Discussion by Fritierkelle** [microsoft/microsoft-ui-xaml#7361] [Q&A] [answered]
Hi,
I've started a new project for testing (WinUI packaged app, SDK stable 1.1) but i can't figure it out. I tried it for the page and for the grid as well as adding the event by code.

I've included 
"winrt/Windows.UI.Xaml.Input.h", 
"winrt/Windows.Foundation.h" and also 
"winrt/Windows.UI.Input.h" 
but i can't resolve it. 

(also on tapped event)

![image](https://user-images.githubusercontent.com/19283500/178480606-a59e75a3-e23a-4de9-a395-1022b1582683.png)

Is it a bug?

**Comment by Fritierkelle** [comment, marked as answer]
#include "winrt/Microsoft.UI.Xaml.Input.h"

**Reply by michael-hawker** [reply]
@Fritierkelle would be good to share how you fixed your issue for others who may find this in the future. Assume you realized you needed the `Microsoft.UI.Xaml.Input` namespace instead?
